tests/cpp/test_host_ir_integration.cpp

```diff
@@ -113,32 +113,6 @@ TEST_F(HostIrIntegrationTest, Sum) {
       "");
 }
 
-TEST_F(HostIrIntegrationTest, Deallocate) {
-  const std::vector<int64_t> sizes = {8, 64};
-  c10::DeviceIndex device_index = 0;
-
-  resetPeakMemoryStats(device_index);
```

Comment:
@nsarka I suspect you need something like https://github.com/NVIDIA/Fuser/blob/ce3d607c0730339c2122c9f42e20cb4fc23f09c5/tests/cpp/test_runtime.cpp#L27-L29